dist/breadcrumbs/ds4/breadcrumbs.css

```diff
@@ -1,6 +1,5 @@
 nav.breadcrumb,
 nav.breadcrumbs {
-  color: #333;
```

Comment:
Hmm, something's not right, we lost the color styles in the css.

Reply:
I don't remember modifying the file at all. Do you think I should just manually add back the colors in dist?

Reply:
In fact, I never touched anything in dist up until then. It seems that modifying the file manually does the trick but I'm seeing if there's a better way.

Reply:
Upon further investigation, running `yarn build` removes the colors in `dist`, even when I manually add them.